doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -853,7 +867,7 @@ macro dlsym(func, lib)
     end
```

Comment:
github moved my comment to a different line again. I was talking about the dlsym macro definition